### ISSUE TITLE
[Item] Scale of the Sands Exalted Reputation Rings have the wrong ICD

### ISSUE THREAD

**Issue by Manaci** [Atlantiss/NetherwingBugtracker#6500]
**Current behaviour**:
[Band of the Eternal Champion](https://db.excalibur.ws/?item=29301) has 60 second internal cooldown. This is probably also the case with the other exalted rings from Scale of the Sands, collectively seen [here](https://db.excalibur.ws/?quest=11105).

**Expected behaviour**: 
[Band of the Eternal Champion](https://db.excalibur.ws/?item=29301) should only have a 45 second internal cooldown, same with the 3 other versions.
[Proof 1](https://www.wowhead.com/item=29301/band-of-the-eternal-championdb#comments) [Proof 2](http://webcache.googleusercontent.com/search?q=cache:nwloCMZatI0J:www.arenajunkies.com/topic/17259-people-who-are-good-at-math-help-me-out/+&cd=16&hl=en&ct=clnk&gl=dk&client=firefox-b-d)

**Server Revision**: 
3213

**Comment by Evolvee** [comment]
this is from your links
FeelsWeirdMan


![image](https://user-images.githubusercontent.com/1942409/71296869-79d55980-2381-11ea-8273-b46cd32537ec.png)
![image](https://user-images.githubusercontent.com/1942409/71296883-85c11b80-2381-11ea-88b1-612042e8bd71.png)
![image](https://user-images.githubusercontent.com/1942409/71296901-940f3780-2381-11ea-8ecb-c1aca79c0f14.png)
![image](https://user-images.githubusercontent.com/1942409/71296937-b4d78d00-2381-11ea-9935-077b8b021827.png)
![image](https://user-images.githubusercontent.com/1942409/71296963-cde03e00-2381-11ea-9a88-7be4d8f2d1c0.png)

maybe u dont count it from when the buff fades off?

**Comment by Manaci** [comment]
@Evolvee that's what I get for reporting for Thunderbad heh. Now he says that the other rings already have a 45 ICD and only the dps one is 60 ICD. That is probably some better proof.

**Comment by Evolvee** [comment]
we need proof from retail, not proof "argumentum a contra rio" of NW -.-

**Comment by Manaci** [comment]
The [Sage Ring] (https://www.wowhead.com/item=29305/band-of-the-eternal-sage#comments) looks to have better proof, also sites that ALL rings has a 45 ICD that starts after the ring buff runs out. Same in the [Restorer Ring](https://www.wowhead.com/item=29309/band-of-the-eternal-restorer#comments). Also how the rings before had a [15 sec duration + 45 sec ICD](https://www.wowhead.com/spell=35083/band-of-the-eternal-sage#comments:id=182116) and then in 2.4 they got their intended 10 sec duration = still 45 sec ICD I presume.

**Comment by Manaci** [comment]
Okay now there has been some more shit from Thunderbad and I should have never begun reporting this from him since I'm not even home monkaS. Now he thought the ICD was when the buff pops and not after it fades like it should be. So the champion ring is apparently 60 sec "ICD" when it pops and the caster and healer ring is only 45 sec "ICD" when they pop. So now it turns out the champion ring has a 50 sec ICD and the caster and healer ring (and possibly the tank ring) have have a lower ICD then intended at only 35 sec  according to my bear Boi. [Proof](https://www.wowhead.com/spell=35083/band-of-the-eternal-sage#comments:id=182116)
This page also says that until 2.4 the buff duration was 15 sec instead of 10 sec for the rings. That means that the current ICD after the buff fades stayed 45 sec and the whole rotation went to 55 sec then. So the real full rotation should be should be 55 sec and not 60 sec but ICD should still be 45 sec then on all rings. Would make sense and why pre 2.4 comments are wrong. 
Anyways this is the last time I write a bug report for someone else.

**Comment by Evolvee** [comment]
tldr: 45 sec from when it falls down=55 sec

ez

**Comment by Godmatik** [comment]
![60s](https://github.com/Atlantiss/NetherwingBugtracker/assets/37913400/c8fdf77d-9f4d-45a7-ad9c-d9e83da49304)

From classic wowhead

So, it seems correct.

Closing.
